docs/Routes.md

```diff
@@ -169,17 +169,39 @@ fastify.get('/', options, async function (request, reply) {
 })
 ```
 
-As you can see we are not calling `reply.send` to send back the data to the user. You just need to return the body and you are done!
+<a name="callback"></a>
+### From callback to promises
```

Comment:
Conceptually, `reply.send()`  is not a callback. It can be used with callback and promise APIs.